instance/models/instance.py

```diff
@@ -217,6 +222,19 @@ class GitHubInstanceQuerySet(models.QuerySet):
     Additional methods for instance querysets
     Also used as the standard manager for the GitHubInstance model
     """
+    def update_or_create_from_pr(self, pr):
```

Comment:
@omarkhan To preserve your refactor here despite my remarks about keeping the choice of domain name in the task, you could add the domain name to this method:

```
def update_or_create_from_pr(self, pr, sub_domain):
```